packages/kotti-ui/source/kotti-style/_comments.scss

```diff
@@ -174,7 +177,7 @@
 
 // KtCommentInput
 
-.comment-input {
+.kt-comment-input {
```

Comment:
dead code

Reply:
i couldn't find this class at all, can u identify where it is

Reply:
That might explain why the comment input looks buggy